libraries/core-react/src/components/Table/DataCell/DataCell.tokens.ts

```diff
@@ -7,6 +7,7 @@ const {
       cell_text: cellTypography,
       cell_numeric_monospaced: cellNumericTypography,
     },
+    _modes: { compact: compactTypography },
```

Comment:
Also add the compact cell_numeric_monospaced, here and in variants below

Reply:
Its not missing as the merge function [here](https://github.com/equinor/design-system/blob/6f82d0f52b5fea513eb2cae4f76b7a2bc3e42d20/libraries/core-react/src/components/Table/DataCell/DataCell.tsx#L23) retains the `fontFeature` thats unique for numeric. But its maybe not to obvious, I can try to see if there is a more transparent way of doing this. Problem is we have multiple permutations with compact now 🤔

Reply:
I have altered the way the variants is applied. Check it out @pomfrida